solutions/swb-reference/src/services/keyPairService.ts

```diff
@@ -14,13 +14,17 @@ import {
   KeyPairPlugin,
   ListKeyPairsRequest,
   SendPublicKeyRequest,
-  SendPublicKeyResponse
+  SendPublicKeyResponse,
+  DatabaseError,
+  NoKeyExistsError,
+  NonUniqueKeyError
 } from '@aws/swb-app';
 import { DynamoDBService, PaginatedResponse } from '@aws/workbench-core-base';
 
 export default class KeyPairService implements KeyPairPlugin {
   private _dynamoDbService: DynamoDBService;
-  private _resourceType: string = 'key-pair';
+  // TODO: have this be 'keyPair' in workbench-core since SWB ref uses ssh key but other apps might use diff keys
```

Comment:
might want to make this a personal note instead of TODO in the app

Reply:
I plan on doing this TODO next week when I extract into workbench-core and don't want to lose it in my personal notes since there isn't actually a ticket to track that effort.